### DISCUSSION TITLE
Packing times wrong! "Search string not found: "/supportedTSExtensions = .*(?=;)/""

### DISCUSSION THREAD

**Discussion by yuhou-cl** [alex8088/electron-vite#678] [Q&A]
![image](https://github.com/user-attachments/assets/3345636c-baf3-46b3-82d0-474896490d76)

**Comment by zzg** [comment]
https://github.com/vuejs/language-tools/issues/5018#issuecomment-2494783497

**Reply by yuhou-cl** [reply]
It has been solved, thank you mahogany！
